server/lobbyconnection.py

```diff
@@ -965,32 +978,35 @@ async def command_game_join(self, message):
             game = self.game_service[uuid]
         except KeyError:
             await self.send({
-                "command": "notice",
+                "command": "game_join_failed",
```

Comment:
For the rest of these, I think we will need to continue sending the notice messages in addition to the `game_join_failed` messages for protocol backwards compatibility. We can mark them as deprecated with a comment though and remove them with a breaking change (protocol version 2).

Reply:
Yes, I have developed a client version check mechanism, so we can remain backwards-compatible.